EXILED.props

```diff
@@ -7,7 +7,7 @@
 
   <PropertyGroup Condition="$(BuildProperties) == '' OR $(BuildProperties) == 'true'">
     <TargetFramework>net48</TargetFramework>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>11.0</LangVersion>
```

Comment:
No

Reply:
Why?

Reply:
> Why?

We need more discussion about updating C# i will not allow if not everyone is okay with it